api/routes/posts.js

```diff
@@ -0,0 +1,124 @@
+const express = require("express");
+const User = require("../models/User");
+const Post = require("../models/post");
+
+const app = express();
+// to get all the posts
+app.get("/posts", (req, res) => {
+  Post.findAll({
+    attributes: ["topic", "discription", "city"],
+    include: [
+      {
+        model: User,
+        as: "user",
+        attributes: ["name", "email"],
+      },
+    ],
+  })
+    .then((data) => {
+      console.log(data);
+      res.status(200).send(data);
+    })
+    .catch((err) => {
+      console.log(err);
+      res.sendStatus(400);
+    });
+});
+
+// to get one persons post
+app.get("/posts/:id", (req, res) => {
+  const user = req.params;
```

Comment:
Is this one specific post or all posts of one specific person @gelilaa  ? If we show all posts of one person in profile page, we may also need such a route @gelilaa

Reply:
it's supposed to be for posts of one specific person. it asks all of one persons post where the the id is user.id.

Reply:
@gelilaa We need  a specific post route too, when the user click on the post, redirect the detailed post page.